Cargo.toml

```diff
@@ -37,12 +37,10 @@ serde_derive = "1.0"
 winapi = { version = "0.3", features = ["sspi", "rpcdce", "impl-default", "timezoneapi", "wincrypt"] }
 url = "2.2.2"
 reqwest = { version = "0.11", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], optional = true, default-features = false }
-picky-krb = "0.3.1"
+picky-krb = { git = "https://github.com/Devolutions/picky-rs.git", rev = "f7dc334" }
```

Comment:
I will update the `picky-krb` dependency once https://github.com/Devolutions/picky-rs/pull/173 is merged and the new `picky-krb` is published.